src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -1,7 +1,7 @@
 ﻿---
 ################################
 # Assembly references report
-# Report date/time: 2024-02-12T15:03:48.2745900Z
```

Comment:
for some reason git created a conflict and didn't fixed itself even after merge from master. I had to manually resolve conflict and this happened.